src/js/components/Menu/Menu.js

```diff
@@ -143,10 +149,12 @@ const Menu = props => {
 
   const onPreviousMenuItem = event => {
     event.preventDefault();
+    const isTab =
```

Comment:
Can we avoid the duplication of this line and line #118, and maybe use a function the accepts event?

Reply:
I think that's a good idea

Reply:
I think that's a good idea.